packages/shared/components/Chip.svelte

```diff
@@ -1,9 +1,10 @@
 <script lang="typescript">
     import { Text } from 'shared/components'
+    import { FontWeightText } from './Text.svelte'
 
     export let label: string = ''
 </script>
 
-<div class="bg-gray-500 dark:bg-gray-700 dark:bg-opacity-20 rounded-full px-2 py-1">
-    <Text type="p" smaller classes="text-white uppercase">{label}</Text>
+<div class="bg-gray-100 dark:bg-gray-700 dark:bg-opacity-20 rounded-full px-3 py-1">
+    <Text type="p" smaller fontWeight={FontWeightText.semibold} color="gray-600" classes="uppercase">{label}</Text>
 </div>
```

Comment:
Ahh just realised I made a generic pill component, and there was a chip component existing already. Could we make this a DeveloperIndicatorPill that wraps a Pill component?